testbed/addons/awslb/construct.ts

```diff
@@ -0,0 +1,47 @@
+import cdk = require('@aws-cdk/core');
+import eks = require('@aws-cdk/aws-eks');
+import iam = require('@aws-cdk/aws-iam');
+import * as yaml from 'js-yaml';
+import * as request from 'sync-request';
+
+export interface AWSLoadBalancerControllerProps {
+    cluster: eks.Cluster;
+    namespace: string;
+}
+
+export class AWSLoadBalancerController extends cdk.Construct {
+    constructor(scope: cdk.Construct, id: string, props: AWSLoadBalancerControllerProps) {
+        super(scope, id);
+
+        const sa = props.cluster.addServiceAccount('aws-lbc', {
+            name: "aws-lbc",
```

Comment:
Is this the standard name?

Reply:
aws docs uses `aws-load-balancer-controller` - https://docs.aws.amazon.com/eks/latest/userguide/aws-load-balancer-controller.html